packages/menu/src/SubMenu/SubMenu.tsx

```diff
@@ -118,10 +135,7 @@ export const SubMenu = InferredPolymorphic<SubMenuProps, 'button'>(
     const handleChevronClick = (e: React.MouseEvent) => {
       // we stop the event from propagating and closing the entire menu
       e.nativeEvent.stopImmediatePropagation();
-
-      if (setOpen) {
-        setOpen(!open);
-      }
+      setOpen(o => !o);
```

Comment:
not in the scope of your changes but was curious, did we intentionally leave out keyboard events for this? I'm seeing I can't toggle the submenu and looks like it only has a click event here

Reply:
No, that's not intentional. I think there's a ticket in this epic to resolve that https://jira.mongodb.org/browse/LG-4120